openshift-client/openshift-client-kubecfg-task.yaml

```diff
@@ -2,19 +2,19 @@ apiVersion: tekton.dev/v1alpha1
 kind: Task
 metadata:
   name: openshift-client-kubecfg
-  namespace: default
 spec:
   inputs:
     resources:
-      - name: clusterdetails
+      - name: cluster
         type: cluster
     params:
-      - name: args
+      - name: ARGS
```

Comment:
nit: I wonder if we need params to be all uppercase :angel:

Reply:
There seems to be a convention for params to be all caps when I checked the rest of them in the catalog

Reply:
fair enough